Updated task.py, but does not affect to solve it. Data were generated by the right way 

更新了task.py，但不影响做题，数据是按照正常的方式生成的
